src/components/BookingBreakdown/LineItemProviderCommissionMaybe.js

```diff
@@ -12,11 +12,7 @@ const { Money } = sdkTypes;
 // Validate the assumption that the commission exists and the amount
 // is zero or negative.
 const isValidCommission = commissionLineItem => {
-  return (
-    commissionLineItem &&
-    commissionLineItem.lineTotal instanceof Money &&
-    commissionLineItem.lineTotal.amount <= 0
-  );
+  return commissionLineItem.lineTotal instanceof Money && commissionLineItem.lineTotal.amount <= 0;
```

Comment:
We should check if the `commissionLineItem` exists before trying to access `lineTotal` or `amount`. It's not clear why this change has been made.